src/content/pages/index.html

```diff
@@ -44,6 +44,22 @@ <h1 class="section-title">{{ site.homepage.title }}</h1>
                     <a href="{{ links.getStarted }}" target="_blank" class="c-btn c-btn--primary c-btn--large">{{ site.shared.get_started }}</a>
                     <a href="{{ links.donate }}" class="c-btn c-btn--secondary c-btn--large donate-link">{{ site.shared.become_a_sponsor }}</a>
                 </div>
+
+                {% set highly_sponsors = sponsors.platinum | concat(sponsors.gold) %}
+
+                <div class="eslint-highly-sponsors">
+                    <div class="sponsor-text">{{ site.homepage.sponsor.text }}</div>
+                    <div class="sponsors">
+                        {% for sponsor in highly_sponsors %}
+                        <a href="{{ sponsor.url | url }}" class="sponsor-link" rel="nofollow noopener sponsored" target="_blank">
+                            <img loading="lazy" decoding="async" src="{{sponsor.image}}" height="128"
+                                alt="{{sponsor.name}}"
+                                title="{{ sponsor.name }}" />
+                        </a>
+                        {% endfor %}
+                        <a href="/sponsors/#silver">{{ site.homepage.sponsor.more }}</a>
```

Comment:
Let's link this to the top of the page. I think that's a more natural landing spot.
```suggestion
                        <a href="/sponsors">{{ site.homepage.sponsor.more }}</a>
```